package.json

```diff
@@ -11,6 +11,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.11.0",
     "react-addons-create-fragment": "^15.6.2",
+    "react-dom": "^16.12.0",
```

Comment:
react-dom doesn't match the version of React 2 lines above. Javascript people: does that matter? (@xlecours ? @HenriRabalais ?)

Reply:
@driusan the version doesn't need to match. Also to note the react we use in LORIS is the latest found on npm and similar with the react-dom.

Reply:
As far as I know, they don't have to match.